edlib/src/edlib.cpp

```diff
@@ -1303,8 +1303,8 @@ static int obtainAlignmentHirschberg(
     int queryIdxLeftStart = max(scoresLeftStartIdx, scoresRightStartIdx - 1);
     int queryIdxLeftEnd = min(scoresLeftStartIdx + scoresLeftLength - 1,
                           scoresRightStartIdx + scoresRightLength - 2);
-    int leftScore, rightScore;
-    int queryIdxLeftAlignment;  // Query/row index of cell in left column where alignment is passing through.
+    int leftScore = 0, rightScore = 0;
```

Comment:
Hm I am pretty sure I did not want to initialize this guys on purpose, because I am certain code below will set them correct values.
If you do need to set them to something, which I guess you are doing because of warnings, let's rather set them to -1, since that is "nonsense" value for them.

Reply:
I'll work on this

Reply:
Done